### ISSUE TITLE
unexpected error error="wrong number of arguments (1 for 3)"

### ISSUE THREAD

**Issue by harmw** [kazegusuri/fluent-plugin-prometheus#4]
When launching fluentd with the exporter plugin installed I get the following error:
```
2016-06-01 22:59:45 +0000 [error]: unexpected error error="wrong number of arguments (1 for 3)"
  2016-06-01 22:59:45 +0000 [error]: /usr/lib/ruby/gems/2.2.0/gems/fluentd-0.14.0.pre.1/lib/fluent/plugin/filter_record_transformer.rb:172:in `prepare_placeholders'
  2016-06-01 22:59:45 +0000 [error]: /usr/lib/ruby/gems/2.2.0/gems/fluent-plugin-prometheus-0.1.2/lib/fluent/plugin/in_prometheus_monitor.rb:20:in `configure'
```
Looks like this is due to a typo in https://github.com/kazegusuri/fluent-plugin-prometheus/blob/master/lib/fluent/plugin/in_prometheus_monitor.rb#L19

*placeholder_expnader* should read *placeholder_expander*

**Comment by harmw** [comment]
Hm, guess the typo is in more places so that wouldn't break anything. Excuse that.

**Comment by kazegusuri** [comment]
Thank you for a filing issue. I heard this plugin doesn't work with fluentd 0.14. I will fix it soon.

**Comment by harmw** [comment]
yea, I'm using 0.14 👍 

Another error, the listener doesn't open up:
```
[warn]: parameter 'port' in <source>
  type prometheus_monitor
  port 24232
  interval 30s
  <labels>
    host ${hostname}
  </labels>
</source> is not used.
```

**Comment by kazegusuri** [comment]
Fixed some errors for v0.14. Actually the error is v0.14 *pre* specific, please use a release version.
And also fixed typo, thanks report.




> Another error, the listener doesn't open up:
> 
> [warn]: parameter 'port' in <source>
>   type prometheus_monitor
>   port 24232
>   interval 30s
>   <labels>
>     host ${hostname}
>   </labels>
> </source> is not used.

The config is wrong. `prometheus_monitor` type doesn't use `port`, only `interval` and labels.
`prometheus_monitor` type is a source of metrics, which obtains metrics from monitor_agent (Fluentd built-in feature).
Use `prometheus` type for a process to serve metrics via http for Prometheus.
